blogposts/2020/can-you-hear-me-no-but-we-read-you.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Can you hear me? No, but we read you—an experiment with video calls"
+author: Quinn Keast
+authorUrl: https://twitter.com/quinnkeast
+publishDate: 2020-10-15
+tags: [blog]
+slug: can-you-hear-me-no-but-we-read-you-an-experiment-with-video-calls
+heroImage: /blog/silent-team-call.png
+published: true
+---
+
+At Sourcegraph, we’re all-remote. That means we write a lot—in emails, Google Docs, Github, and Slack. We work hard to cultivate a written-first culture.
+
+And that works beautifully for me—because, you see, I have a hearing problem. Or as I like to say, I’m deaf as a post.
```

Comment:
Now I'm thinking, maybe:

```suggestion
And that works beautifully for me. Because, you see, I have a hearing problem. Or as I like to say, I’m deaf as a post.
```